plugins/meta/iplink/README.md

```diff
@@ -0,0 +1,54 @@
+# iplink plugin
+
+## Overview
+
+This plugin changes some interface attributes, like 'ip link' command.
+It does not create any network interfaces and therefore does not bring connectivity by itself.
+This iplink plugin applies changes to interfaces created by previously applied plugins as meta plugin.
```

Comment:
It can be used in a chain to apply several specific changes to interface previously configured by other plugin.

Reply:
Fixed